app/modules.go

```diff
@@ -341,6 +341,26 @@ func versionedStoreKeys() map[uint64][]string {
 			stakingtypes.StoreKey,
 			upgradetypes.StoreKey,
 		},
+		v3: {
```

Comment:
[no change needed] verified this is the exact same list as v2 https://www.diffchecker.com/pHyajXyz/